### PR TITLE
Viewer - Replace NetBarcode to BarcodeLib, add FieldBlock

### PR DESCRIPTION
**Pull request by tinohager**
The other library would support significantly more barcodes and is available in a stable version on nuget.

https://www.nuget.org/packages/BarcodeLib

### CODE REVIEW

**Comment by YipingRuan** [comment]
I assume nobody will actually use this viewer to print things. (People will still send ZPL command to the printer).
So anything can draw the correct size and location would be enough, what do you say?

**Comment by tinohager** [comment]
@YipingRuan Well, only if the barcode is interpreted correctly, the size on the label is correct. That's why I think it makes sense to reproduce everything as well as possible.

File: src/BinaryKits.Zpl.Viewer/ElementDrawers/Interleaved2of5BarcodeDrawer.cs [hunk 1]
```diff
@@ -25,21 +26,35 @@ public override void Draw(ZplElementBase element)
                     y -= barcode.Height;
                 }
 
-                //TODO: Change to an other project that supports Interleaved2of5 Barcode
-                var barcodeElement = new Barcode();
-                barcodeElement.Configure(new BarcodeSettings
+                var barcodeElement = new BarcodeLib.Barcode();
```
Comment (YipingRuan) [inline src/BinaryKits.Zpl.Viewer/ElementDrawers/Interleaved2of5BarcodeDrawer.cs:29]: Instead of using the namespace?

**Comment by tinohager** [comment]
Remove dependency to NetBarcode
https://github.com/BinaryKits/BinaryKits.Zpl/issues/49

**Comment by YipingRuan** [comment]
https://github.com/BinaryKits/BinaryKits.Zpl/issues/49

**Pull request merged**